src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -69,8 +69,7 @@ define(function (require, exports, module) {
     };
 
     ParameterEditor.prototype._setSpaces = function (parameters) {
-        this._spaces = [];
-        this._spaces.push(parameters.search(/\S/));
+        this._spaces = [parameters.search(/\S/)];
```

Comment:
technical nit: this code does not seem to do what its naming implies...  what was the intention of this function?